test/gtest/common/test_obj_size.cc

```diff
@@ -57,7 +57,7 @@ UCS_TEST_F(test_obj_size, size) {
     EXPECTED_SIZE(ucp_rkey_t, 32);
 #endif
     /* TODO reduce request size to 240 or less after removing old protocols state */
-    EXPECTED_SIZE(ucp_request_t, 264);
+    EXPECTED_SIZE(ucp_request_t, 280);
```

Comment:
I think we can consider increasing number of lanes to 32 instead of 64. Or allowing creation of 64 lanes only for wireup and restrict the max lanes number by 32 or 16 for RNDV protocols. What sounds better for you?

Reply:
both sounds ok. Maybe it is better to restrict protocol;s to use up to 16 lanes? Then no need to keep large maps in the request

Reply:
pr #9862 should address it

Reply:
#9862 is not enough, also #9864 is required

Reply:
request size was rolled back.